src/util/slack.ts

```diff
@@ -0,0 +1,52 @@
+
+import { IncomingWebhook } from '@slack/webhook';
+import * as constants from '../common/constants';
+import Logger from '../common/logger';
+
+const log = Logger.createLogger('util.slack');
+
+const webhook = new IncomingWebhook(constants.SLACK_WEBHOOK_URL);
+
+type levelType = 'error' | 'info' | 'debug';
```

Comment:
Type의 이름은 PascalCase를 따라야 합니다.

```suggestion
type LevelType = 'error' | 'info' | 'debug';
```